misc/cleanup-dash.py

```diff
@@ -399,6 +399,11 @@ def add_annotation(dashboard):
     user_input = raw_input(prompt)
     if user_input:
         if user_input == 'Yes':
+            active_variables = [tag]
+            for templating in dashboard['templating']['list']:
+                if templating['type'] == "query" and templating['hide'] == 0:
```

Comment:
```suggestion
                if templating['type'] == 'query' and templating['hide'] == 0:
```

Better to be consistent with `'` and `"`.